src/test/java/visualizations/LineChartTest.java

```diff
@@ -0,0 +1,41 @@
+package visualizations;
+
+import dataloader.CsvDataLoader;
+import org.junit.Test;
+import tech.tablesaw.api.Table;
+
+import java.io.File;
+import java.io.IOException;
+
+public class LineChartTest {
+
+    /**
+     * Test Scenario: The user provides a data set
+     * and wants to plot a line chart.
+     * Expected Output: The chart is plotted correctly.
+     */
+    @Test
```

Comment:
This goes for all the test I run for these classes. I am getting these warnings: 
`SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".`
`SLF4J: Defaulting to no-operation (NOP) logger implementation`
`SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.`
They link [here](https://www.slf4j.org/codes.html#StaticLoggerBinder). 
Any ideas?

Reply:
Currently we will leave it as is. The internals of Tablesaw seems to use the SLF4j but we need to specify where this is located. If you throw it in the CLASSPATH it should go away I believe

Reply:
added slf4j to classpath.

Reply:
problem not solved yet although slf4j is in classpath.

Reply:
Ok let's skip this at the moment. I will create an issue to have it in our TODOs